### DISCUSSION TITLE
Process community rewards cycle: 7/02 -> 7/29

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#187] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):
```
# NOTE ASA Stats Community rewards 7/02 - 7/29
@aunnaemeka 0.03 damo
Abhimanyu 0.09 damo
AlgoKnight 0.03 damo
AlgoRhythMatic 0.51 damo
Damo 0.24 damo
DragMZ 0.03 damo
moloch10 0.33 damo
prince2486 0.03 damo
RandomTask 0.06 damo
SCN9A 0.06 damo
tkpremier 0.03 damo
u/Aromatic-Ad3922 0.03 damo
UncleDooom 0.03 damo
TOTAL 1.5 damo
```

Please review and comment if there are any questions or concerns. This period uses a 3x coefficient to multiply measurements from the [Rewards Model](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=1130085233) tab from our spreadsheet.

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
Great job, as always, we're so happy to have you AlgoRhythm!! :)
